docusaurus/docs/Android/04-guides/05-working-with-offline.md

```diff
@@ -0,0 +1,186 @@
+# Working with Offline
+
+When using the offline library, you first have to build a `ChatDomain` instance as described on the [Getting Started](../01-basics/03-getting-started.md#creating-a-chatdomain) page.
+
+`ChatDomain` is a singleton, just like `ChatClient`. This means that you'll always have just one instance of it in your application. After building it, you can access it with the `instance` method:
+
+```kotlin
+val chatDomain = ChatDomain.instance()
+```
+
+`ChatDomain` is user specific, so you'll want to disconnect your user and create a new `ChatClient` and `ChatDomain` when the user in your app changes. You can do this using `ChatClient`, which will also disconnect `ChatDomain` automatically:
+
+```kotlin
+ChatClient.instance().disconnect()
+```
+
+### Flows or LiveData
+
+There are two different `ChatDomain` interfaces available in the offline SDK, in different packages:
+
+- `*.offline.ChatDomain`: Exposes its state as `StateFlow` objects you can collect. You can use this if you're using Kotlin coroutines in your app. 
+- `*.livedata.ChatDomain`: Exposes state as `LiveData` objects. Use this if you're writing Java code or prefer LiveData to Flows.
+
+:::note
+Make sure you build the same version of `ChatDomain` that you're using in your code elsewhere, otherwise it might not be initialized.
+:::
+
+If you want to use both Flows and LiveData, only initialize the LiveData-based `ChatDomain`, as that will also initialize the Flow-based one internally.
+
+### Retry Policy
+
+By default, the retry policy for the ChatDomain is set to retry three times and wait `attempt * 1000` milliseconds between attempts. It will also retry when the connection recovers. You can set your own `RetryPolicy` like this:
```

Comment:
```suggestion
By default, the retry policy for the `ChatDomain` is set to retry three times and wait `attempt * 1000` milliseconds between attempts. It will also retry when the connection recovers. You can set your own `RetryPolicy` like this:
```